Description: support building with Sphinx 1.2
 This reverts upstream changeset 1c1cd1eac7ce.
 .
 When Sphinx 1.3 reaches unstable, this patch should be removed
 together with the hack in debian/rules.
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-needed
Last-Update: 2015-06-24

--- a/doc/sphinx/conf.py
+++ b/doc/sphinx/conf.py
@@ -91,7 +91,7 @@
 
 # The theme to use for HTML and HTML Help pages.  Major themes that come with
 # Sphinx are currently 'default' and 'sphinxdoc'.
-html_theme = 'classic'
+html_theme = 'default'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
